packages/http/src/forwarder/HttpForwarder.ts

```diff
@@ -32,6 +38,22 @@ export class HttpForwarder
     };
   }
 
+  private updateHostHeaders(baseUrl: string, headers?: IHttpNameValue) {
+    // no headers? do nothing
+    if (!headers) return headers;
+
+    // host header provided? override with actual hostname
+    if (headers.hasOwnProperty('host')) {
+      return {
+        ...headers,
+        host: urijs(baseUrl).host(),
+        forwarded: `host=${headers.host}`,
```

Comment:
Shouldn't this be capitalised?
```suggestion
        Forwarded: `host=${headers.host}`,
```

Reply:
No necessarily - http headers are case insensitive. I made it lowercase because it looks better when constructing JS object - and just because of that.